packages/client/test/relayClient.test.ts

```diff
@@ -212,55 +239,56 @@ describe("user behaviour", () => {
   });
 
   describe("editing the listing", () => {
-    let itemId: Uint8Array;
-    // create item
-    beforeEach(async () => {
-      const metadata = new TextEncoder().encode(
-        JSON.stringify({
-          name: "test",
-          description: "test",
-          image: "https://http.cat/images/200.jpg",
-        }),
-      );
-      itemId = await relayClient.createItem({
-        price: "10.99",
+    let tagId = { raw: objectId() };
+    beforeAll(async () => {
+      const metadata = {
+        title: "test",
+        description: "test",
+        images: ["https://http.cat/images/200.jpg"],
+      };
+      const price = priceToUint256("12.99");
+
+      const requestId = await relayClient.listing({
+        id: tagId,
+        price: {
+          raw: price,
+        },
         metadata,
       });
-      expect(itemId).not.toBeNull();
-    });
-
-    test("update item - price", async () => {
-      await relayClient.updateItem({ itemId, price: "20.99" });
-      expect(itemId).not.toBeNull();
+      expect(requestId).not.toBeNull();
     });
 
     test("update item - metadata", async () => {
-      await relayClient.updateItem({
-        itemId,
-        metadata: new TextEncoder().encode(
-          JSON.stringify({
-            name: "new name",
-            image: "https://http.cat/images/200.jpg",
-          }),
-        ),
+      const requestId = await relayClient.updateListing({
+        id: tagId,
+        metadata: {
+          title: "new name",
+          images: ["https://http.cat/images/200.jpg"],
+        },
       });
-      expect(itemId).not.toBeNull();
+      expect(requestId).not.toBeNull();
     });
-
-    describe("tagging", () => {
-      let tagId: Uint8Array;
-      beforeEach(async () => {
-        tagId = await relayClient.createTag({ name: "Testing New Tag" });
-        expect(tagId).not.toBeNull();
-      });
-      test("add item to tag", async () => {
-        await relayClient.updateTag({ tagId, addItemId: itemId });
-      });
-      test("remove item from tag", async () => {
-        await relayClient.updateTag({ tagId, removeItemId: itemId });
+    test("#handlePingRequest - relay does not end connection", async () => {
```

Comment:
I don't feel confident that this is actually testing what we want it to... Trying to test that pingRequest is correctly responded to so relay doesnt end connection